packages/material-ui/src/Popper/Popper.d.ts

```diff
@@ -48,8 +48,7 @@ export interface PopperProps extends Omit<React.HTMLAttributes<HTMLDivElement>,
    */
   container?: PortalProps['container'];
   /**
-   * Disable the portal behavior.
-   * The children stay within it's parent DOM hierarchy.
+   * The `children` content will be inside the DOM hierarchy of the parent component.
```

Comment:
Wondering whether this should just be:

```suggestion
   * The `children` will be inside the DOM hierarchy of the parent component.
```